resources/lib/uzg.py

```diff
@@ -70,41 +71,21 @@ def get_ondertitel(self, whatson_id):
             
         def get_play_url(self, whatson_id):
             ##token aanvragen
-            data = self.__get_data_from_url('http://ida.omroep.nl/npoplayer/i.js')
-            token = re.compile('.token\s*=\s*"(.*?)"', re.DOTALL + re.IGNORECASE).search(str(data)).group(1)
+            data = self.__get_data_from_url('http://ida.omroep.nl/app.php/auth')
+            token = re.search(r'token\":"(.*?)"', data).group(1)
             ##video lokatie aanvragen
-            data = self.__get_data_from_url('http://ida.omroep.nl/odi/?prid='+whatson_id+'&puboptions=adaptive&adaptive=yes&part=1&token='+self.__get_newtoken(token))
+            data = self.__get_data_from_url('http://ida.omroep.nl/app.php/'+whatson_id+'?adaptive&adaptive=yes&part=1&token='+token)
+	    xbmc.log("plugin.video.uzg:: data %s" % data)
```

Comment:
Prima, wat is eigenlijk dan nu de officiële manier om iets te loggen in je plugin? Dat kon ik niet zo snel vinden. Voor het debuggen had ik dit even nodig.